docs/developers_guide/micro_frontends_in_open_edx.rst

```diff
@@ -16,35 +16,168 @@ To run an MFE locally you must install `node 12 <https://nodejs.org>`_ (with npm
+
+``LOGIN_URL``
+  Example: ``https://courses.example.com/login``
+
+  The fully-qualified URL to the login page in the target environment.
+
+``LOGOUT_URL``
+  Example: ``https://courses.example.com/logout``
+
+  The fully-qualified URL to the API endpoint in the target environment which performs a user logout.
+
+``STUDIO_BASE_URL``
+  Example: ``https://studio.example.com``
+
+  The fully-qualified URL of Studio in the target environment. This is often used by micro-frontends from which educators may wish to go edit course content, such as `frontend-app-learning <https://github.com/edx/frontend-app-learning>`_
+
+``MARKETING_SITE_BASE_URL``
+  Example: ``https://www.example.com``
+
+  The fully-qualified URL of the environment's marketing site.  Today this is used by frontend-app-account to link to a demographics collection page, and also as a base URL for optimizely experiment scripts.  **This is deprecated, and will be removed in a future release.** Use cases in frontend-app-account will be refactored to use different environment variables, since neither use case has anything to do with marketing.
+
+``ORDER_HISTORY_URL``
+  Example: ``https://orders.example.com``
+
+  The fully-qualified URL to the Order History page.  This is often used in a user menu in the header of micro-frontends.
+
+Analytics
+---------
+
+``SEGMENT_KEY``
+  Example: ``ABCDEFGHIJKLMNOPQRSTUVWXYZ123456``
+
+  An implementation-specific environment variable which, when configured, enables integration with Segment, a user analytics and data management vendor. 32 characters.
+
+Authentication
+--------------
+
+``ACCESS_TOKEN_COOKIE_NAME``
+  Example: ``example-jwt-cookie-header-payload``
+
+  The name of a cookie where JWT authentication data should be stored.
+
+``USER_INFO_COOKIE_NAME``
+  Example: ``example-user-info``
+
+  The name of the cookie where user-specific information should be stored.
+
+``REFRESH_ACCESS_TOKEN_ENDPOINT``
+  Example: ``https://courses.example.com/login_refresh``
+
+  The fully-qualified URL of the endpoint which allows the micro-frontend to refresh a user's JWT authentication.  The inconsistency in naming with other fully-qualified URL environment variables is unfortunate.
+
+``CSRF_TOKEN_API_PATH``
+  Example: ``/csrf/api/v1/token``
+
+  The path of the service's CSRF token API endpoint.  This gets prefixed with the domain of a request requiring CSRF protection.  This token is added to the X-CSRFToken header on outgoing requests.
+
+Branding
+--------
+
+``SITE_NAME``
+  Example: ``Your Site Name Here``
+
+  The user-facing name of the site.
+
+``LOGO_URL``
+  Example: ``https://edx-cdn.org/v3/default/logo.svg``
+
+  The fully-qualified URL of the site's logo file suitable for use in the application's header.
+
+``LOGO_TRADEMARK_URL``
+  Example: ``https://edx-cdn.org/v3/default/logo-trademark.svg``
+
+  The fully-qualified URL of the site's logo with a trademark in the image suitable for use in a footer.
```

Comment:
The trademark isn't required, right? I might rephrase as `The fully-qualified URL of the site's logo suitable for use in a footer. This is often a logo with a trademark such as (R) or (TM).`